src/custom/components/Menu/index.tsx

```diff
@@ -14,7 +14,10 @@ export const StyledMenu = styled(MenuMod)`
   }
 `
 
-const Policy = styled(InternalMenuItem)`
+const Policy = styled(InternalMenuItem).attrs(attrs => ({
+  ...attrs,
+  target: '_blank'
+}))`
```

Comment:
Yep, I red the comment, but problem is that this unrelated change might add additional unrelated discussion. 

For example, in this case, I don't think we should add this. Why do we want to open the policies in a blank page? Is this a requirement coming from somewhere?

Reply:
David mentioned this somewhere (don't remember where). To not loose state/avoid over complicate state handling.
If that's loaded in the same page while one action is pending, the state will be lost.

Reply:
Mm, i don't understand. It feels strange to me to open a _blank page to the same app. 
If they navigate, its fine to lose the state

I'm probably missing sth

Reply:
I think as David points out, between the trade-off of keeping state vs. navigating (and priorities..) I think a target=_blank is an option for now. Just make sure to do the `target="_blank" rel="noopener"` for security.

Reply:
let's discuss this in subsequent PR(s)